### PR TITLE
added a new page for Presence

### PR DESCRIPTION
**Pull request by pk-pranshu**
## Description

Added a new page for the new Presence APIs and supported scenarios

### CODE REVIEW

**Comment by pk-pranshu** [comment]
@jason-ha - Need your help write out the following section - 
LatestValueControl, NotificationsValueManager and onboarding sample code

File: docs/content/docs/build/presence.md
[the remark this answers is not in the document]
Comment (pk-pranshu) [reply docs/content/docs/build/presence.md:1]: Yes it is! Didn't have any new content to add as such. The README was pretty well written to include all concepts and sample code. If there is a way to get that directly that's fine too.

[the remark this answers is not in the document]
Comment (Josmithr) [reply docs/content/docs/build/presence.md:1]: The mechanism is the same as before - using markdown-magic to embed contents into `.md` files. I have work scheduled to look into patterns that work with the newly supported `.mdx` files, but that hasn't landed yet.

[the remark this answers is not in the document]
Comment (Josmithr) [reply docs/content/docs/build/presence.md:1]: Also note that the Docusaurus PR has now been merged. You will want to update this PR with the new contents from main and verify the new page is in the right place :)

File: docs/content/docs/build/presence.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+The key scenarios that the new Presence APIs are suitable for includes:
+- User presence
+- Collaborative cursors
+- Notifications
+
+
+## Concepts
+
+A session is a period of time when one or more clients are connected to a Fluid service. Session data and messages may be exchanged among clients, but will disappear once the no clients remain. (More specifically once no clients remain that have acquired the session `IPresence` interface.) Once fully implemented, no client will require container write permissions to use Presence features.
```
Comment (Rick-Kirkham) [inline docs/content/docs/build/presence.md:24]: As I understand it, each client has its own session with the service. But it looks like you are talking about some kind of shared session that multiple clients are part of. If that's the case, let's make sure this new sense of "session" is distinct. I recommend we use the term "presence session" or even "shared presence session" or "common presence session" whenever we're talking about session in this new sense.

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: docs/docs/build/presence.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+---
+title: Presence
```
Comment (Josmithr) [inline docs/docs/build/presence.md:2]: You'll also want to specify `sidebar_position: <number>` to control the ordering for how this appears in the sidebar.

**Review by jason-ha — APPROVED** [review]
Needs cleanup - but as first page it is okay

File: docs/docs/build/presence.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+Latest map value manager retains the most recent atomic value each attendee has shared under arbitrary keys. Values associated with a key may be nullified (appears as deleted). Use `LatestMap` to add one to `PresenceStates` workspace.
+
+#### NotificationsManager
+
+Notifications value managers are special case where no data is retained during a session and all interactions appear as events that are sent and received. Notifications value managers may be mixed into a `PresenceStates` workspace for convenience. They are the only type of value managers permitted in a `PresenceNotifications` workspace. Use `Notifications` to add one to `PresenceNotifications` or `PresenceStates` workspace.
+
+## Onboarding
+
+** Needs update**
```
[the remark this answers is not in the document]
Comment (pk-pranshu) [reply docs/docs/build/presence.md]: updated

File: docs/content/docs/build/presence.md (hunk 1 above)
Comment (pk-pranshu) [reply docs/content/docs/build/presence.md]: Will change in the next iteration

**Review by Rick-Kirkham — APPROVED** [review]

**Pull request merged**